### ISSUE TITLE
CreateBlock method gets called multiple times during

### ISSUE THREAD

**Issue by cesartl** [AdityaGupta1/minecraft-catapult#17]
If you put a logging line in `createBlock` method, you will see that it gets called multiple time. This could potentially affect performance on low specification laptop.

I believe the issue is in this code:

```
public void onLivingUpdate() {
		super.onLivingUpdate();
		this.setPosition(this.lastTickPosX, this.lastTickPosY, this.lastTickPosZ);

		for (Trajectory trajectory : trajectories) {
			this.worldObj.spawnEntityInWorld(createBlock(trajectory.getAngle(), trajectory.getPower(), true,
					trajectory.getColor()));
		}
	}
```

Do we really need to call the `CreateBlock` method in the `onLivingUpdate` method?

**Comment by AdityaGupta1** [comment]
Yes, the reason it needs to be called multiple times is so the trajectory keeps showing. The particles I use disappear after a while so I need to constantly create blocks to add new particles.

I could make it spawn them less often though.
